coding-the-legislation/10_basic_example.md

```diff
@@ -61,6 +61,54 @@ To make sure that the formula you have just written works the way you expect, yo
 
 You can check the [YAML tests documentation](writing_yaml_tests.md) to learn more about how to write YAML tests, and how to run them.
 
+## Example with Enum variable
+
+Let's say we have a `HOUSING_OCCUPANCY_STATUS` enumeration:
+
+```py
+HOUSING_OCCUPANCY_STATUS = Enum([
+    u'Tenant',
+    u'Owner',
+    u'Free lodger',
+    u'Homeless'])
+```
+
+To declare an enumeration variable `housing_occupancy_status` of `HOUSING_OCCUPANCY_STATUS` in your legislation, the `EnumCol` type should be defined as follow:
+
+```py
+class housing_occupancy_status(Variable):
+    column = EnumCol(
+        enum = HOUSING_OCCUPANCY_STATUS
+        )
+    entity = Household
+    definition_period = MONTH
+    label = u"Legal housing situation of the household concerning their main residence"
+```
+
+A default value could also be added:
+```py
+    column = EnumCol(
+        enum = HOUSING_OCCUPANCY_STATUS,
+        default = 1
```

Comment:
Explain what this value is. Nowhere have I written `1`, what does this mean? How can I define my own default value?